docker/bookkeeper/entrypoint.sh

```diff
@@ -74,6 +75,27 @@ create_bookie_dirs() {
   done
 }
 
+# Create a Bookie ID if this is a newly added bookkeeper pod
+# or read the Bookie ID if it already exists
+set_bookieid() {
+  IFS=',' read -ra directories <<< $1
+  BK_ID_DIR="${directories[0]}"
+  BK_ID_FILE="${BK_ID_DIR}/id"
+  if [ `find ${BK_ID_FILE} | wc -l` -gt 0 ]; then
+    echo "Found BookieID"
+    BK_bookieId=`cat ${BK_ID_FILE}`
+  else
```

Comment:
Instead of looking for the options, cant we look if there are legers/journals metadata, if they are present then it is an upgrade case. If not it is the new pod coming up.

Reply:
Have incorporated this change